Feature/task list component

Vartanchik opened the pull request:
List that contains tasks for current day.

Vartanchik committed:
add TaskListComponent

Vartanchik committed:
add OnInit interface implementation

Vartanchik committed:
remove empty constructor

Vartanchik committed:
merge develop into feature/task_list_component

Anatoliy-F reviewed (changes requested):
There are a few remarks:
1. Layout
I think that the message about the lack of tasks should be centered vertically
![image](https://user-images.githubusercontent.com/34834746/213862980-344f1a16-545a-4501-8054-af45f277ab9f.png)

2. task-list.component.html row 12
You init taskList variable with empty array, check "taskList === undefined" is unnecessary

3. task-list.component.html rows 33-42, 54-62, 86-97
The same code is repeated three times, I suggest doing the markup once.

4. task-list.component.ts
Instead hardcoding test data in ngOnInit() and settings (initionalCountOfTasksForShow) in component variables, I propose create TaskService

Sihuta reviewed (commented):
In addition to @Anatoliy-F's review, I'd like to mention a few points:

1. The text on task items should be white since the background is green.
2. The time allocated for tasks should be in the same unit of measure as pomodoros (i.e. in minutes).
3. Tasks completed on 100% should be moved to the end of the list.
4. The list of tasks might be long enough that you must scroll the whole page to see them. I suggest making scrollable only the list component. In this case, the main page will look better than this:
![image](https://user-images.githubusercontent.com/63560452/213874762-32ad5607-e3a5-4811-b229-d3651f643415.png)

Vartanchik committed:
Refactor task-list-component:
- centered message about empty list;
- remove "taskList === undefined" checking;
- remove repeatable code in html.

Vartanchik committed:
create TaskService and add getTasks method

Vartanchik committed:
remove showAllTasks method

Anatoliy-F reviewed (approved):
Good job

Vartanchik committed:
add white text on green background

Vartanchik committed:
add task-list scrolling

Vartanchik committed:
move done tasks to lists end

Vartanchik committed:
change allocated time format to HH:mm:ss
